internal/pkg/externalplugins/cherrypicker/cherrypicker.go

```diff
@@ -576,6 +577,7 @@ func (s *Server) handle(logger *logrus.Entry, requestor string,
 				return nil
 			}
 		} else {
+			hasConflicts = true
```

Comment:
We should not take whether the `options.IssueOnConflict` option is turned on as one of the conditions to judge whether there is a conflict.

Reply:
This option will return directly when turned on?

Reply:
Whether the `hasconflicts = true` statement in the else code block will bring trouble to people's understanding?